### DISCUSSION TITLE
make_scanned

### DISCUSSION THREAD

**Discussion by joaqueendex** [ImageMagick/ImageMagick#7334] [Help]
### ImageMagick version

7.1.1

### Operating system, version and so on

Windows 10 22H2 build 19045.4412

### Description

hi everyone

I need a tool - a standalone desktop app - that helps to create 'scanned' copies of PDF files. The source files may contain different contents: text, images, etc. They can also be of different sizes, number of pages, be coloured or black and white. For clarity here's an example of a service that does this: https://makescanned.com/

With this command, I managed to get pretty good black and white copies:

magick convert -density 300 "F:\PDFFolder\Source.pdf" -alpha remove -rotate 0.33 -attenuate 0.15 +noise Multiplicative +repage -monochrome -compress group4 "F:\PDFFolder\Output.pdf"

But when I tried to create a colour copy by removing the -monochrome option from the above command, I got an output file several times larger than the original.

Thanks in advance for any tips on how to reduce the size of the colour final file.

**Comment by snibgo** [comment]
If your source file is vector, the rasterization (conversion to pixels) will greatly increase the size.

Your operations that rotates and adds noise will increase the "information" content, so reduce the ability to compress the image. You might reduce the number of colours.

If you want more specific advice, please supply a sample input, and a desired output.
